OWML.ModHelper.Interaction/ModInteraction.cs

```diff
@@ -0,0 +1,72 @@
+﻿using OWML.Common;
+using OWML.ModHelper.Events;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace OWML.ModHelper.Interaction
+{
+    public class ModInteraction : IModInteraction
+    {
+        private readonly List<ModBehaviour> _modList = new List<ModBehaviour>();
+        private readonly IModFinder _finder;
+        private readonly Dictionary<IModData, List<IModData>> dependantDict = new Dictionary<IModData, List<IModData>>();
+
+        public ModInteraction(List<ModBehaviour> list, IModFinder finder)
+        {
+            _modList = list;
+            _finder = finder;
+
+            foreach (var mod in _finder.GetMods())
+            {
+                List<IModData> temp = new List<IModData>();
+                foreach (var mod2 in _finder.GetMods())
```

Comment:
Double loop - can probably be simplified. Commenting so I can come back to it.

Reply:
Nah, double loop is fine.

But variable names can be improved. Suggestion:

- temp -> dependants
- mod2 -> otherMod